test/Microsoft.NET.Build.Tests/EvaluatorFastPathTests.cs

```diff
@@ -0,0 +1,63 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.NET.Build.Tests
+{
+    public class EvaluatorFastPathTests : SdkTest
+    {
+        public EvaluatorFastPathTests(ITestOutputHelper log) : base(log)
+        {
+
+        }
+
+        [Fact]
+        public void FastPathDoesNotNeedReflection()
+        {
+            var testAsset = _testAssetsManager
+                .CopyTestAsset("MSBuildBareBonesProject")
+                .WithSource();
+            var command = new MSBuildCommand(testAsset, string.Empty);
+            command
+                .WithEnvironmentVariable("MSBuildLogPropertyFunctionsRequiringReflection", "true")
+                .WithWorkingDirectory(testAsset.Path);
+            command
+                .ExecuteWithoutRestore()
+                .Should()
+                .Pass();
+
+            var logPath = Path.Combine(testAsset.Path, "PropertyFunctionsRequiringReflection");
+            File.Exists(logPath).Should().BeFalse();
+        }
+
+        [Theory]
+        [InlineData("console")]
+        [InlineData("webapp")]
+        public void EnsureDotnetCommonProjectPropertyFunctionsOnFastPath(string alias)
```

Comment:
```suggestion
        public void EnsureDotnetCommonProjectPropertyFunctionsOnFastPath(string alias_)
```
`alias` is a keyword so let's avoid it as a name for variable